tests/helpers/destroy-app.js

```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember';
+import { cleanup } from 'ember-wait-for-test-helper/wait-for';
 
 export default function destroyApp(application) {
+  cleanup();
```

Comment:
After thinking about this a bit more, we don't need to make `cleanup` the default behavior. Instead of using the `afterInstall` blueprint hook we could just say in the documentation to add this line to `destroy-app.js` if you're interested in ensuring cleanup. I was thinking it would be a nice turn-on-by-default feature, but since it's not necessary maybe we can avoid. Not sure, on the fence.